src/encoded/schemas/target.json

```diff
@@ -16,7 +16,7 @@
     ],
     "properties": {
         "schema_version": {
-            "default": "8"
+            "default": "9"
```

Comment:
We need to remove organism from target and change all references of target.organism to target.gene.organism.

Reply:
target.gene_name should be target.gene.symbol.       I am not sure how we reconcile target.label and target.gene -- but I guess this was always an issue (and one of the reasons we don't allow non-admins to submit)

Reply:
As mentioned last week, this will be step 2 & 3 like this:
1. Get this gene schema and target schema in this or one release.
2. Post thousands of gene data/object with the following two copies: "target.gene_name" -> "gene.symbol" and "target.dbxref" -> "gene.dbxref"
3. Patch targets with gene embedding.
4. In a next release, replace "gene_name" and "dbxref" with "gene.symbol" and "gene.dbxref" target schema. Pop/delete "gene_name" and "dbxref" from target data/object using upgrade.

Reply:
Agreed, I was wrong about the order.